charts/latest/azurefile-csi-driver/values.yaml

```diff
@@ -23,6 +23,9 @@ image:
     repository: mcr.microsoft.com/oss/kubernetes-csi/csi-node-driver-registrar
     tag: v1.2.0
     pullPolicy: IfNotPresent
+## Name of secrets (present in namespace) containing Docker credentials used to pull images
```

Comment:
this is where we put default values, so it's not necessary to add comments here since by default it's empty

Reply:
@fredgate are you going to fix this? just remove the comments is ok